lockbox-iosTests/ItemListPresenterSpec.swift

```diff
@@ -563,8 +576,18 @@ class ItemListPresenterSpec: QuickSpec {
 
             describe("sortingButton") {
                 beforeEach {
-                    self.subject.onViewReady()
-                    self.view.fakeOnSortingButtonPressed.onNext(())
+                    let voidObservable = self.scheduler.createColdObservable([next(50, ())])
+
+                    voidObservable
+                            .bind(to: self.subject.sortingButtonObserver)
```

Comment:
@fluffyemily I tried merging `master` in to get this branch up to the latest. But I can't tell how to reconcile this test.

The changes from 4 days ago at #762 mean that `self.subject.sortingButtonObserver` doesn't quite exist here anymore and I couldn't figure this out myself.

Reply:
It should be just a case of keeping 

```
let itemSettingObservable: TestableObservable<Setting.ItemListSort> = self.scheduler.createColdObservable([next(50, Setting.ItemListSort.alphabetically)])
                    itemSettingObservable
                        .bind(to: self.subject.listSortedObserver)
                        .disposed(by: self.disposeBag)

                    self.scheduler.start()
```

and losing the other changes.

Reply:
Actually, this change causes all of my tests to fail as the fake button doesn't cause the `ScrollAction` to be fired. I may have to rethink these tests.